src/main/scala/com/typesafe/sbt/PackagerPlugin.scala

```diff
@@ -73,16 +73,16 @@ object SbtNativePackager extends AutoPlugin {
 
     import SettingsHelper._
 
-    @deprecated("Use enablePlugins(xxxDeployPlugin)", "1.x")
-    def deploymentSettings =
-      makeDeploymentSettings(Debian, Debian / packageBin, "deb") ++
-        makeDeploymentSettings(Rpm, Rpm / packageBin, "rpm") ++
-        makeDeploymentSettings(Windows, Windows / packageBin, "msi") ++
-        makeDeploymentSettings(Universal, Universal / packageBin, "zip") ++
-        addPackage(Universal, Universal / packageZipTarball, "tgz") ++
-        makeDeploymentSettings(UniversalDocs, UniversalDocs / packageBin, "zip") ++
-        addPackage(UniversalDocs, UniversalDocs / packageXzTarball, "txz") ++
-        makeDeploymentSettings(Debian, Debian / genChanges, "changes")
+    // @deprecated("Use enablePlugins(xxxDeployPlugin)", "1.x")
+    // def deploymentSettings =
+    //   makeDeploymentSettings(Debian, Debian / packageBin, "deb") ++
+    //     makeDeploymentSettings(Rpm, Rpm / packageBin, "rpm") ++
+    //     makeDeploymentSettings(Windows, Windows / packageBin, "msi") ++
+    //     makeDeploymentSettings(Universal, Universal / packageBin, "zip") ++
+    //     addPackage(Universal, Universal / packageZipTarball, "tgz") ++
+    //     makeDeploymentSettings(UniversalDocs, UniversalDocs / packageBin, "zip") ++
+    //     addPackage(UniversalDocs, UniversalDocs / packageXzTarball, "txz") ++
+    //     makeDeploymentSettings(Debian, Debian / genChanges, "changes")
```

Comment:
shall this be uncommented or deleted?

Reply:
Removed in bdeb29f1b23cb58a19eb89e670cc25c218427160